pkg/deploy-manager/arbiter.go

```diff
@@ -0,0 +1,67 @@
+package deploymanager
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// electArbiterZone is a helper function just for the tests. In real deployments, the arbiter zone will be picked by the user.
+// In our tests, we will just pick the zone with the least number of nodes as the arbiter zone.
+func (t *DeployManager) electArbiterZone() (arbiterZoneElect string, err error) {
+	nodes, err := t.k8sClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{LabelSelector: "node-role.kubernetes.io/worker"})
```

Comment:
Might there be a zone with only a master and no worker? The common arbiter scenario is to have just the master node in the arbiter zone and no worker, but maybe that won't apply in the clusters where this test will run.

Reply:
Yes, I am making an assumption for the deploy-manager that the arbiter pods will be deployed on a worker node. This is to keep the deploy manager simple. The placement algorithm in the ocs-operator now has the master node toleration set for the arbiter pods.